token_classification/README.MD

```diff
@@ -0,0 +1,40 @@
+## Token Classification service
+
+Token Classification using Transformers on AWS Lambda. Check root readme for complete setup info.
+
+## Request format
+
+```
+{
+        "texts": ["Mark is going back to Germany from South Africa", "John Adams is performing live in Venezuela"],
+        "model_name": "dslim/bert-base-NER",
+        "tokenizer_name": "dslim/bert-base-NER"
```

Comment:
If these 2 are not passed, it should use the default values

Reply:
yes, it will do, this is the readme file. Do you want me to add a comment?

Reply:
You can write `# optional` in readme

Reply:
cool